build.gradle.kts

```diff
@@ -26,6 +26,12 @@ allOpen {
     annotation("jakarta.persistence.MappedSuperclass")
 }
 
+tasks.test {
+    jvmArgs(
+        "--add-opens", "java.base/java.time=ALL-UNNAMED",
```

Comment:
?

Reply:
https://mockk.io/doc/md/jdk16-access-exceptions.html

Reply:
for å kunne statisk mocke LocalDate.now

https://mockk.io/doc/md/jdk16-access-exceptions.html